evalml/tests/automl_tests/test_iterative_algorithm.py

```diff
@@ -283,17 +283,18 @@ def test_iterative_algorithm_stacked_ensemble_n_jobs_regression(n_jobs, linear_r
     assert seen_ensemble
 
 
-@pytest.mark.parametrize("parameters", [1, "hello", 1.3, -1.0006, [1, 3, 4], (2, 3, 4)])
```

Comment:
I think we should still keep a list example so that we can verify the IterativeAlgorithm now handles them properly!

Reply:
@freddyaboulton Correct me if I'm misunderstanding but the idea now is that we *don't* support lists so they shouldn't be handled correctly / are expected to error, right? 😅

Reply:
I'm thinking about the Drop Columns case for the data check actions. We would specify the columns to drop in the `pipeline_params` of the `IterativeAlgorithm` (at least that's the impl in #1981). So we can still pass lists it's just that the `IterativeAlgorithm` shouldn't do anything to that list and just pass it to the pipeline in the parameters dict.